sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1287,10 +1287,12 @@ int gnrc_netif_ipv6_add_prefix(gnrc_netif_t *netif,
         IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_MULTIHOP_P6C) &&
         gnrc_netif_is_6ln(netif)) {
 
-        /* configure compression context */
-        if (gnrc_sixlowpan_ctx_update_6ctx(pfx, pfx_len, valid)) {
-            DEBUG("gnrc_netif: add compression context for prefix %s/%u\n",
-                   ipv6_addr_to_str(addr_str, pfx, sizeof(addr_str)), pfx_len);
+        if (!IS_ACTIVE(CONFIG_GNRC_NETIF_IPV6_DO_NOT_COMP_PREFIX)) {
```

Comment:
Just some bikeshedding: Aren't positive (enabled by default) options preferable when possible? 

```suggestion
        if (IS_ACTIVE(CONFIG_GNRC_NETIF_IPV6_AUTO_6CTX)) {
```

Reply:
I tried that, but then the option wasn't included at all for some reason. Maybe @leandrolanzieri can provide any inside on that?

Reply:
I think you have to always define it in a header/source file - Kconfig does not run by default

Reply:
> […] the option wasn't included at all for some reason

I meant the option was not visible in `make menuconfig` when saying "wasn't included", so this is not a problem of header file defintitions.

Reply:
> Just some bikeshedding: Aren't positive (enabled by default) options preferable when possible?

The problem we had with doing so, is that Kconfig boolean symbols are not defined when set to `n` (pretty much like a CFLAG), so if you use Kconfig, set it to `n`, then the header file will override it (unless you check for Kconfig being used).

> I meant the option was not visible in `make menuconfig` when saying "wasn't included", so this is not a problem of header file defintitions.

can you post the snippet that was not working?

Reply:
Not sure what the problem was, but now it worked as desired with @benpicco's probosal. Adapted accordingly.